src/ActionMailerNext.Standalone/ActionMailerNext.Standalone.csproj

```diff
@@ -1,116 +1,24 @@
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Newtonsoft.Json.6.0.8\lib\net45\Newtonsoft.Json.dll</HintPath>
-    </Reference>
-    <Reference Include="PreMailer.Net">
-      <HintPath>..\packages\PreMailer.Net.1.3.0\lib\net40\PreMailer.Net.dll</HintPath>
-    </Reference>
-    <Reference Include="RazorEngine">
-      <HintPath>..\packages\RazorEngine.3.4.0\lib\net45\RazorEngine.dll</HintPath>
-    </Reference>
-    <Reference Include="RestSharp, Version=105.0.1.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\RestSharp.105.0.1\lib\net4\RestSharp.dll</HintPath>
-    </Reference>
-    <Reference Include="SendGrid.SmtpApi">
-      <HintPath>..\packages\SendGrid.SmtpApi.1.2.1\lib\net40\SendGrid.SmtpApi.dll</HintPath>
-    </Reference>
-    <Reference Include="SendGridMail">
-      <HintPath>..\packages\Sendgrid.5.1.0\lib\SendGridMail.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.ComponentModel.DataAnnotations" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Web" />
-    <Reference Include="System.Web.Razor, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\packages\Microsoft.AspNet.Razor.3.0.0\lib\net45\System.Web.Razor.dll</HintPath>
-    </Reference>
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Helpers\ExtendedTemplateBase.cs" />
-    <Compile Include="Helpers\StandaloneUrlHelpers.cs" />
-    <Compile Include="Helpers\UtilHelper.cs" />
-    <Compile Include="Helpers\ViewSettings.cs" />
-    <Compile Include="Helpers\NameValueCollectionExtensions.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="RazorEmailResult.cs" />
-    <Compile Include="RazorMailerBase.cs" />
-    <Compile Include="Helpers\ExtendedTemplateResolver.cs" />
-    <Compile Include="Helpers\StandaloneHtmlHelpers.cs" />
-    <Compile Include="TemplateResolvingException.cs" />
+    <PackageReference Include="Handlebars.Net" Version="2.0.9" />
   </ItemGroup>
+
   <ItemGroup>
-    <None Include="ActionMailerNext.Standalone.nuspec" />
-    <None Include="app.config" />
-    <None Include="packages.config">
-      <SubType>Designer</SubType>
-    </None>
+    <ProjectReference Include="..\ActionMailerNext\ActionMailerNext.csproj" />
   </ItemGroup>
+
   <ItemGroup>
-    <ProjectReference Include="..\ActionMailerNext\ActionMailerNext.csproj">
-      <Project>{85ec0023-cbbf-4c91-8681-d88da6334305}</Project>
-      <Name>ActionMailerNext</Name>
-    </ProjectReference>
+    <Reference Include="System.ComponentModel.DataAnnotations">
+      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.7.2\System.ComponentModel.DataAnnotations.dll</HintPath>
```

Comment:
This is not gonna work out of your folder structure. Looks like you have to install this package in .NET standard https://www.nuget.org/packages/System.ComponentModel.Annotations/